spec/models/organisation_spec.rb

```diff
@@ -5,7 +5,7 @@
     it { expect(subject).to validate_presence_of :name }
     it { expect(subject).to validate_presence_of :slug }
     it { expect(subject).to validate_presence_of :organisation_type }
-    it { expect(subject).to allow_value("call_centre").for(:organisation_type) }
+    it { expect(subject).to allow_value("drs_call_center").for(:organisation_type) }
     it { expect(subject).to_not allow_value("doesnt_exist").for(:organisation_type) }
```

Comment:
for line 8 and 9 you can use `validates_inclusion_of` from `Shoulda`. Regardless of that, the test will feel quite close to the implementation itself.